### PR TITLE
Added subject in events, committees and citizen projects contact form

### PR DESCRIPTION
**Pull request by thomasbisignani**

### CODE REVIEW

File: src/CitizenProject/CitizenProjectContactActorsCommand.php [hunk 1]
```diff
@@ -36,6 +42,16 @@ public function getSender(): Adherent
         return $this->sender;
     }
 
+    public function setSubject(?string $subject)
```
Comment (Remg) [inline src/CitizenProject/CitizenProjectContactActorsCommand.php]: missing return type

File: src/Committee/CommitteeContactMembersCommand.php [hunk 2]
```diff
@@ -36,6 +42,16 @@ public function getSender(): Adherent
         return $this->sender;
     }
 
+    public function setSubject(?string $subject)
```
Comment (Remg) [inline src/Committee/CommitteeContactMembersCommand.php]: missing return type

File: src/Mailer/Message/CitizenProjectContactActorsMessage.php [hunk 3]
```diff
@@ -23,7 +24,7 @@ public static function create(array $recipients, Adherent $host, string $content
             '275088',
             $first->getEmailAddress(),
             $first->getFullName(),
-            'Vous avez un nouveau message d\'un porteur de projet',
+            sprintf('[Projet citoyen] %s', $subject),
```
Comment (Remg) [inline src/Mailer/Message/CitizenProjectContactActorsMessage.php]: `"[Projet citoyen] $subject"`

File: src/CitizenProject/CitizenProjectContactActorsCommand.php (hunk 1 above)
Comment (thomasbisignani) [reply src/CitizenProject/CitizenProjectContactActorsCommand.php]: :ok_hand: :ok_man:

**Pull request merged**